src/pages/keep-safe/index.js

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import styled from 'styled-components'
 import SEO from '../../components/containers/seo'
 import { SecurityIconGrid } from './_icon-grid'
+import Show from 'components/containers/show'
```

Comment:
you can run npm run test, to fix all of import orders

Reply:
Sure